include/librepr/type/name.h

```diff
@@ -74,14 +75,27 @@ struct TemplateInfo<U<Ts...>> {
     }(std::make_index_sequence<argument_count>());
   }
 };
-
 }  // namespace detail
 
 template <typename T>
 char const* get_mangled_name() {
-#if defined(_WIN32)
-  return typeid(T).raw_name();
+#if USING(REPR_MSVC)
+  // Just in case...
+  if constexpr(detail::msvc::has_rawname) {
+    static auto& ty = typeid(T);
+    // .name() must be called since we have lazy symbol
+    // loading enabled. Otherwise when accessing later
+    // ._UndecoratedName will be NULL.
+    static volatile auto _ = ty.name();
```

Comment:
oof. Can you give it some name to stop clang-tidy from complaining though? :P

Minimum length is set to 3 characters I think